### PR TITLE
countdown: new title field, description made optional

### PR DESCRIPTION
**Pull request by MaximilianKrambach**
caveat: old countdowns will disappear from the database in the migration

### CODE REVIEW

File: client/src/app/site/projector/components/countdown-list/countdown-list.component.html [hunk 1]
```diff
@@ -50,48 +59,72 @@ <h2 translate>Countdowns</h2>
...
                 </mat-form-field>
-            </p><p>
+            </p>
+            <p>
+                <mat-form-field>
+                    <input
+                        formControlName="description"
+                        matInput
+                        placeholder="{{ 'Description' | translate }}"
+                        required
```
Comment (FinnStutzenstein) [inline client/src/app/site/projector/components/countdown-list/countdown-list.component.html]: Please remove

**Comment by MaximilianKrambach** [comment]
This countdown surprises me. And as the title can be changed by the user, translating a countdown title does not seem to be an useful idea. I'll just default-name this one 'countdown 1'

**Comment by MaximilianKrambach** [comment]
done @FinnStutzenstein

**Comment by FinnStutzenstein** [comment]
The translation is needed for all values, that are hardcoded in the server. We also need to translate every projector name, because the "defaultprojector" must be translated, regardless, if the user can change the name or not.

**Comment by MaximilianKrambach** [comment]
Okay, I named it "Default countdown" and put a translate in everywhere the title is used.

File: openslides/core/migrations/0019_remove_old_countdowns.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Generated by Django 2.1.5 on 2019-02-27 11:17
```
Comment (FinnStutzenstein) [inline openslides/core/migrations/0019_remove_old_countdowns.py]: Please name logically connecte migration files with a prefix `_1`, `_2`, ...
-> `countdown_title_{1,2}.py`

**Comment by MaximilianKrambach** [comment]
file renamed, rebased

**Comment by emanuelschuetze** [comment]
@FinnStutzenstein Please approve

**Pull request merged**